src/Http/JsonMetaRequest.php

```
@@ -91,7 +91,7 @@ protected function createRequest()
     {
         $request = $this
         ->request
-        ->withMethod('get')
+        ->withMethod('GET')
```

Perhaps we should just be transparent and throw an `InvalidArgumentException` if the request method given is not `GET`. The default has always been `GET`